tools/bsn_proposal_generation.py

```diff
@@ -169,6 +169,10 @@ def main():
     print('Begin Proposal Generation Module')
     args = parse_args()
     cfg = mmcv.Config.fromfile(args.config)
+    # import modules from string list.
+    if cfg.get('custom_imports', None):
```

Comment:
repeating these three lines are so verbose. the mechanism should be improved.

### Proposal

add a new function `mmcv.Config.resolve_custom_imports`, which imports modules into `mmcv.Config.custom_imports`.
add an option in `mmcv.Config.from_file` to enable `resolve_custom_imports`.

After this, things will be:

```python
cfg = mmcv.Config.fromfile(args.config, resolve_custom_imports=True) # could be shorter if True is default
from mmcv.Config.custom_imports import *
```

Benefit: reduce 3 lines to 1 line

cc. @hellock

Reply:
or incorporate `import_modules_from_strings` into fromfile, and reduce 3 lines to 0 lines.